spring-boot-legacy/src/main/java/org/springframework/boot/legacy/context/web/AnnotationConfigNonEmbeddedWebApplicationContext.java

```diff
@@ -72,19 +82,34 @@ public AnnotationConfigNonEmbeddedWebApplicationContext() {
 	 * Create a new {@link AnnotationConfigNonEmbeddedWebApplicationContext}, deriving
 	 * bean definitions from the given annotated classes and automatically refreshing the
 	 * context.
-	 * @param annotatedClasses one or more annotated classes, e.g. {@link Configuration
-	 * &#64;Configuration} classes
+	 * @param annotatedClasses one or more annotated classes, e.g. {@code @Configuration}
+	 * classes
 	 */
 	public AnnotationConfigNonEmbeddedWebApplicationContext(
-			Class<?>... annotatedClasses) {
+			Class<?>... annotatedClasses)
+	{
```

Comment:
Not a big deal really, but we generally don't put opening braces on a new line in Spring projects.

Reply:
Will fix, yeah that was my normal projects code style sneaking in.